audeer/core/io.py

```diff
@@ -420,14 +428,15 @@ def list_file_names(
         list of path(s) to file(s)
```

Comment:
Maybe we should add here that the list is sorted?

Reply:
Changed to:

![image](https://user-images.githubusercontent.com/10383417/217280985-b2ba5a04-e925-47c8-8ce2-9833a23b89d8.png)